R/shared-methods.R

```diff
@@ -98,15 +98,16 @@ initialise_dataclass <- function(self, level = "1", regions,
 
   if (!is.null(self$level_data_urls[[self$level]])) {
     if (length(self$common_data_urls) > 0) {
-      self$data_urls <- merge(
+      self$data_urls <- unlist(merge(
```

Comment:
What a nightmare - good spot. I think we also want to reorder the merge so common > shared.